src/lcs.nim

```diff
@@ -29,18 +28,23 @@ proc runMain() =
   if licenseType == "":
     # exit cli if type is not set
     echo "License type flag required! (--type, -t)"
-    quit(0)
+    quit(1)
 
-  try:
-    license = licenses[licenseType]
-  except IndexDefect:
+  # Get license contents
+  var license = licenses.getOrDefault(licenseType, "")
+  # If there's no such license
+  if license == "":
     echo "Unknown license type: ", licenseType
-    quit(0)
+    quit(1)
 
 
   # replace license var contents
-  license = license.replace("[fullname]", author).replace("[year]",
-      year).replace("[project]", projectName).replace("[projecturl]", projectUrl)
+  license = license.multiReplace({
```

Comment:
Instead of doing multiple `replace`s you can use https://nim-lang.org/docs/strutils.html#multiReplace%2Cstring%2Cvarargs%5B%5D which allows for a very neat replacing syntax!